public/app/features/users/TokenRevokedModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button, Modal } from '@grafana/ui';
+import config from 'app/core/config';
+
+interface Props {}
+
+interface State {}
+
+export class TokenRevokedModal extends React.Component<Props, State> {
+  redirectToLogin() {
+    window.location.href = config.appSubUrl + '/';
+  }
+
+  render() {
+    return (
+      <Modal title="Token revoked" isOpen={true} onDismiss={this.redirectToLogin} onClickBackdrop={() => {}}>
+        <p>Your token has been revoked because you have reached the maximum amount of concurrent sessions.</p>
+        <p>Please, log in again.</p>
+        <div className="gf-form-group">
+          <Button size="md" variant="primary" onClick={this.redirectToLogin}>
```

Comment:
I think this should be a `LinkButton` component, which uses `a` element which uses `config.appSubUrl + '/'` as an `href`, instead of using JS to do a redirect.

Reply:
I did a quick test by using the `LinkButton` with no `href` attribute and it didn't work. Also did a quick test by providing the `href` attribute `${config.appSubUrl}/}` but it neither worked because we actually need a full page reload.

Additionally, I'll need to keep the `redirectToLogin` method anyway to be called `onDismiss`. So, is it acceptable for you to keep the `Button` with the JS function to perform the redirect + full page reload?

Thanks!! 🙏🏻

Reply:
ah, ok I see. If you need the full page reload then current solution is good 👍